draft-scenarios/coops/close_to_enemies.tex

```diff
@@ -0,0 +1,222 @@
+% !TeX spellcheck = en_US
+\addscenariosection{1}{Cooperative Scenario}{Close to enemies}{\images/ballistics.png}
+
+\begin{multicols}{2}
+
+\textbf{Author:} Kelghor
+
+\textbf{Version:} 1.6
+
+\textbf{Source:} \href{https://discord.com/channels/740870068178649108/1313651865648369725}{Archon Studio Discord}
+
+\textit{``Sir! The raiders are at our gates!''}
+
+\textit{``Sound the warhorn! We'll drive them back as always!'' you command, and go to defend the city. Just as you have done many times before. Such is the harsh life of the border clans, where wars and skirmishes with neighboring rivals never truly end. Fortunately, your alliance remains strong, and you know your back is covered.}
+
+\textit{The time has come to end these long-standing wars and crush your enemies like a bug once and for all!}
+% Translator info: In Homam IV was Quote like: "Rufux will crush this army like a bug." If has scounting and has bigger army. So "like a bug" is little easter egg referencing on this.
+
+\subsection*{\MakeUppercase{Scenario Length}}
+
+This Scenario is played over 7 Rounds.
+
+\subsection*{\MakeUppercase{Player Setup}}
+
+\textbf{Player Count:} 1--6
+
+\textbf{Starting Resources:} 14 \svg{gold}, 5 \svg{building_materials}, 2 \svg{valuables}
+
+\textbf{Starting Income:} 10 \svg{gold}, 2 \svg{building_materials}, 0 \svg{valuables}
+
+\textbf{Starting Units:}
+\begin{itemize}
+  \item  1 × A Few \svgunit{bronze} Units of your choice
+\end{itemize}
+
+\textbf{Town Buildings:} \svgunit{bronze} Dwelling.
+
+\subsection*{\MakeUppercase{Map Setup}}
+
+Take the following Map Tiles and arrange them as shown in the Scenario map layout.
+
+\begin{itemize}
+  \item P × Starting (I) Map Tile
+  \item P × Far (II--III) Map Tile with the Settlement Field of the same terrain as player's Town.
+  \item 3P × Far (II--III) Map Tile without the Settlement Field. Each player takes three Map Tiles of their Faction's terrain with different mines. Each player place two of them on the game board so that they correspond with the map template and are linked together. Each player keeps the last Tile for personal exploration.
+\end{itemize}
+
+Place Far (II--III) Map Tiles with Settlements that match your Faction's terrain, ensuring they align with the positions on the map represented by colored Cubes. Each pair of same-colored Cubes represents a Map Tile (I) and its corresponding Far (II--III) Map Tile with a Settlement.
+Each player place
+
+\textit{Optional: If you want to spice up the game and deepen the cooperation between players, you can take all Far (II--III) Map Tile without the Settlement Field and place them on map randomly. Create a shared pool for discovery from the remaining Tiles.}
+
```

Comment:
OK, I had a thought about this and came up with the following. I think `Map Tile Pool` should not be omitted. The whole random/non-random rule is a place for just that. Then, info in the `Map Setup` can be shortened.

It boils down to describing the map setup in the map setup only, and all the things with map tile pool, either shared/random or not, in the map tile pool. Wanted to make it short and sweet, without too much text. I hope it captures the idea well. What do you think?

```suggestion
\textbf{Map Tile Pool:} Each player takes 1 random Far (II--III) Map Tile.
These Tiles must not contain Settlement Fields.

\textbf{Alternative Pool Setup (Optional):} To spice up the game and deepen cooperation between players, gather all Far (II--III) Map Tiles without Settlement Fields into a shared pool for exploration.

\subsection*{\MakeUppercase{Map Setup}}

Take the following Map Tiles and arrange them as shown in the Scenario map layout:

\begin{itemize}
 \item \textbf{P × Starting (I) Map Tile}
 
 \item \textbf{P × Far (II--III) Map Tile with Settlement}
 Place these Tiles according to the colored cube pairs shown on the map template.
 Each pair indicates positions for a Starting Tile and its matching Settlement Tile.
 
 \item \textbf{2P × Far (II--III) Map Tile without Settlement}
 Players place their two Tiles on designated spots.
 These Tiles must be connected to each other.
\end{itemize}
```

Reply:
I don't know where to start.

The current version seems unnecessarily overcomplicated. The fact that the Map tile pool has to be mentioned in its own section is unnecessarily constraining when it's an alternative method.
However, right now it's extra iteratively imprecise. 

"To spice up the game and deepen cooperation between players, gather all Far (II--III) Map Tiles without Settlement Fields into a shared pool for exploration."
Is not accrue. real deepen of cooperation is not in due shared map tile pool, but due not symmetrical distribution of mines. 
- It can easily happen that one player will have gold mines and other players will have crystals and stones, so then they have to either travel with the pieces or send resources to each other.

The crucial difference is that each player will either use their faction map-tiles or not. 
It's really about factional distribution of pieces, since each faction has its mines the same. It's about aesthetics and also making players less need to co-opt each other.


So either just have randomly distributed tiles in a map-template and a shared pool (needing more co-op)
Or for aesthetics/simplification -> for newbies for example. They each have their own color in their part of the territory in the map-template + 1 of their own tiles in hand to discover...
So each player takes 4 of his map tiles with one set element, and has 3 pieces left. Two he puts on the map in the map-template and the last one he doesn't want to discover. (So wherever he goes there will be waiting for him a possible undiscovered piece , with the mine that the player will be collecting.

I don't think it's necessary to explain the impact of the alternative fire. But rather to write that one is more coherent to cooperation and the other less so.


To me, it might be best to have map-tile pool + mapsetup as one variant, AND then the discovery section repeated for the other variant. 

Which variant will be the alternative doesn't really matter. 

Why am I sticking with the fractional map-tile? Because the 1st one will contain all the information (correct terrain color and content down). It can be more complicated to cast, I'd probably rather put that as a hint/tip or add the code of the map-tiles that belong to each faction in the notes.

One last thing wording this - I'm not sure if it's overkill.
" These Tiles must be connected to each other."

When I translate it, it looks to me like all tiles must be connected to each other, i.e. A+B, B+C but also A+C. However, it is enough if A+B,B+C,C+D are connected, but there is no need for C+A or D+A.

- I have no idea how to write it.


Take bych to rád nějak zjendodušil a víc rozdělil ty dvě možnosti.  Aby byl map pool a map setup víc provázané. 

Osobně sekce map-pool mi přidje strašně divná a nechápu proč je to vlastní sekce, když to jendoznačně patří do map-setupu. Ale to je fail už originálích scénářů.


PS: I Wish marry xmas. :-)